tests/tracking/context/test_databricks_command_context.py

```diff
@@ -0,0 +1,23 @@
+from unittest import mock
+
+from mlflow.utils.mlflow_tags import MLFLOW_DATABRICKS_NOTEBOOK_COMMAND_ID
+from mlflow.tracking.context.databricks_command_context import DatabricksCommandRunContext
+
+
+def test_databricks_command_run_context_in_context():
+    with mock.patch("mlflow.utils.databricks_utils.get_job_group_id") as job_group_id_mock:
+        assert DatabricksCommandRunContext().in_context() == (
+            job_group_id_mock.return_value is not None
+        )
```

Comment:
```suggestion
    with mock.patch("mlflow.utils.databricks_utils.get_job_group_id", return_value="1") as job_group_id_mock:
        assert DatabricksCommandRunContext().in_context()
```

Can we explicitly specify `return_value` here?